.genignore

```diff
@@ -0,0 +1,2 @@
+internal/provider/connection_resource.go
```

Comment:
Since these files have been manually modified, we need to exclude them from generation.